scripts/exgdas_global_marine_analysis_vrfy.py

```diff
@@ -22,189 +22,147 @@
 import gen_eva_obs_yaml
 import marine_eva_post
 import diag_statistics
-from soca_vrfy import plot_config, plot_horizontal_slice, plot_zonal_slice
+from soca_vrfy import StatePlotter, plot_config
 import subprocess
 from datetime import datetime, timedelta
 
 comout = os.getenv('COM_OCEAN_ANALYSIS')
 com_ice_history = os.getenv('COM_ICE_HISTORY_PREV')
 com_ocean_history = os.getenv('COM_OCEAN_HISTORY_PREV')
-data = os.getenv('DATA')
-pdy = os.getenv('PDY')
 cyc = os.getenv('cyc')
 RUN = os.getenv('RUN')
-bcyc = str((int(cyc) - 3) % 24).zfill(2)
 gcyc = str((int(cyc) - 6) % 24).zfill(2)
-gcdate = datetime.strptime(os.getenv('PDY')+os.getenv('cyc'), '%Y%m%d%H') - timedelta(hours=int(os.getenv('assim_freq')))
 
+bcyc = str((int(cyc) - 3) % 24).zfill(2)
 grid_file = os.path.join(comout, f'{RUN}.t'+bcyc+'z.ocngrid.nc')
 
 # for eva
 diagdir = os.path.join(comout, 'diags')
 HOMEgfs = os.getenv('HOMEgfs')
 
-
 #######################################
-# INCREMENT
+# ocean increment
 #######################################
-incr_cmap = 'RdBu'
+
 data_file = os.path.join(comout, f'{RUN}.t'+cyc+'z.ocninc.nc')
 config = plot_config(grid_file=grid_file,
                      data_file=data_file,
-                     colormap=incr_cmap,
+                     lats=np.arange(-60, 60, 10),
+                     variables_zonal=['Temp', 'Salt'],
+                     variables_horiz=['Temp', 'Salt', 'ave_ssh'],
+                     allbounds={'Temp': [-0.5, 0.5],
+                                'Salt': [-0.1, 0.1],
+                                'ave_ssh': [-0.1, 0.1]},
+                     colormap='RdBu',
                      comout=os.path.join(comout, 'vrfy', 'incr'))
+OcnIncPlotter = StatePlotter(config)
```

Comment:
if you are going to use camel case style of naming stuff, variables should start with a lower case.